2power.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
```

Comment:
Nice improvement from last time! Your code compiled! 🎉 You forgot to include 2^0 = 1 and instead of continue, you should have have written "break;" so that your program would end. It got caught in the loop.